### ISSUE TITLE
Adding width to card breaks megafilter

### ISSUE THREAD

**Issue by LorenzoSapora** [dcasia/nova-mega-filter#20]
When adding the nova core method `->width('')` to the megafilter, it breaks. This is purely a CSS problem. It adds `card-panel` to the second div.

It breaks:

- Dropdowns
- Alignment
- Padding
- Margin

:-1: 

![image](https://user-images.githubusercontent.com/25519274/72441658-f7369300-37a2-11ea-8cf2-27f894ac3d49.png)

With `->width()`:
```html
<div class="flex flex-wrap -mx-3 mb-3"><div class="px-3 mb-6 w-3/4"><div class="card mega-filter flex flex-col p-6 card-panel" lens="">
```

Without `->width()`:
```html
<div class="flex flex-wrap -mx-3 mb-3"><div class="px-3 mb-6 w-full"><div class="card mega-filter flex flex-col p-6" lens="">
```

Manually removing it, fixes all issues.

I'm trying to think of a way of fixing this, so I can make a PR for it.

**Comment by LorenzoSapora** [comment]
Could `.mega-filter{height:auto!important}` on https://github.com/dcasia/nova-mega-filter/blob/master/resources/js/components/MegaFilter.vue

Although that's a bit messy. I don't like to `!important` unless I really have to.

**Comment by milewski** [comment]
Your PR fixed this issue, published as version `v0.2.4`
